database/sql/V039__rename_order_class_enum.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE OrderClass RENAME VALUE 'ordinary' TO 'market'
```

Comment:
Have we renamed an enum before? We should double check it really changes all existing values.